src/components/TokenAllowance.jsx

```diff
@@ -98,30 +98,8 @@ class TokenAllowance extends Component {
                   <span>
                     {
                       this.props.system.skr.tapApproved === -1
-                      ? 'Loading...'
```

Comment:
@b-pmcg this shouldn't be removed as you still need to give allowances for wiping. Only the `Shut` text should be deleted.